cypress/support/fragments/eholdings/eHoldingsPackages.js

```diff
@@ -273,6 +276,26 @@ export default {
     cy.do([searchField.fillIn(searchValue), searchButton.click()]);
   },
 
+  titlesSearchFilter: (searchParameter, searchValue, selectionStatus = 'All') => {
+    cy.expect(searchIcon.exists());
+    cy.wait(2000);
+    cy.do([searchIcon.click()]);
+
+    cy.do([chooseParameterField.choose(searchParameter), searchField.fillIn(searchValue)]);
+    // tags
+    // sort options
+    // selection status parameter
```

Comment:
I added those in case there is another testcase that needs to add that information, this to avoid another function. I can remove them if you think is best